test/integration/archive/ArchiveViewer.e2e.test.js

```diff
@@ -53,4 +56,30 @@ describe('Archive Viewer', () => {
         cy.getByTestId('search-input').type('vido');
         cy.contains('Video (Normal).mp4');
     });
+
+    it('Should reset scroll when navigating folders', () => {
+        cy.showPreview(token, fileIdLong);
+
+        cy.getByTitle('Collection').within(() => {
+            cy.get('button').click();
+        });
+
+        cy.get('.ReactVirtualized__Table__Grid').then($tables => {
+            const table = $tables[0];
+            table.scrollTo(0, 50);
+
+            const tableTop = table.getBoundingClientRect().top;
+            cy.get('.ReactVirtualized__Table__row').then($rows => {
```

Comment:
Any way we can avoid using CSS classes here?

Reply:
Is using filename (like `cy.contains('File001.txt')`) better?

Reply:
I think so, yeah, since it's based on user-visible information. We do it in other tests, too.